src/transform-array.js

```diff
@@ -1,6 +1,44 @@
 const CustomError = require("../extensions/custom-error");
 
-module.exports = function transform(/* arr */) {
-  throw new CustomError('Not implemented');
-  // remove line with error and write your code here
+module.exports = function transform(arrayToBeTransformed) {
+  if(!Array.isArray(arrayToBeTransformed)){
+    throw new CustomError('It is not array!');
+  }
+  let result_array = arrayToBeTransformed;
+
+  for (let i=0; i<arrayToBeTransformed.length; i++){
+    if(typeof(arrayToBeTransformed[i]) == 'string'){
```

Comment:
you don't need to nest 'if' here without 'else'. Just use 'break' if condition is true

Reply:
ready for review